recipes/google-cloud-cpp/all/googleapis-grpc-libraries.cmake

```diff
@@ -0,0 +1,267 @@
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/monitoring/metricsscope/v1"
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/monitoring/v3"
+    )
+googleapis_grpc_proto_library(iam_v1_options_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/iam/v1/options.proto"
+    )
+googleapis_grpc_proto_library(iam_v1_iam_policy_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/iam/v1/iam_policy.proto"
+    )
+googleapis_grpc_proto_library(type_calendar_period_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/calendar_period.proto"
+    )
+googleapis_grpc_proto_library(type_color_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/color.proto"
+    )
+googleapis_grpc_proto_library(type_date_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/date.proto"
+    )
+googleapis_grpc_proto_library(type_datetime_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/datetime.proto"
+    )
+googleapis_grpc_proto_library(type_dayofweek_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/dayofweek.proto"
+    )
+googleapis_grpc_proto_library(type_expr_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/expr.proto"
+    )
+googleapis_grpc_proto_library(type_fraction_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/fraction.proto"
+    )
+googleapis_grpc_proto_library(type_interval_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/interval.proto"
+    )
+googleapis_grpc_proto_library(type_latlng_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/latlng.proto"
+    )
+googleapis_grpc_proto_library(type_localized_text_protos
+    PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/localized_text.proto"
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/localized_text.proto"
+    )
+googleapis_grpc_proto_library(type_money_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/money.proto"
+    )
+googleapis_grpc_proto_library(type_month_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/month.proto"
+    )
+googleapis_grpc_proto_library(type_phone_number_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/phone_number.proto"
+    )
+googleapis_grpc_proto_library(type_postal_address_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/postal_address.proto"
+    )
+googleapis_grpc_proto_library(type_quaternion_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/quaternion.proto"
+    )
+googleapis_grpc_proto_library(type_timeofday_protos
+    GRPC_PROTOS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/type/timeofday.proto"
+    )
+googleapis_grpc_proto_library(cloud_dialogflow_v2_protos
+    GRPC_PROTOS_DIRS
+        "${CONAN_GOOGLEAPIS_PROTOS}/google/cloud/dialogflow/v2"
+    )
```

Comment:
All these libraries don't make too much sense to me. They contain [just one proto](https://github.com/googleapis/google-cloud-cpp/blob/a2a34b9cd5b4025e8ca9d68b140db8abe71fc54e/external/googleapis/CMakeLists.txt#L245)! 🤷